[0m12:47:44.445241 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E2691B5EA0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E26B925660>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E26B925420>]}


============================== 12:47:44.453921 | a35811fe-91bc-483e-b0f1-48dd084c5dbf ==============================
[0m12:47:44.453921 [info ] [MainThread]: Running with dbt=1.5.0
[0m12:47:44.453921 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\HP\\.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'send_anonymous_usage_stats': 'True'}
[0m12:48:04.931430 [debug] [MainThread]: Starter project path: C:\Users\HP\OneDrive\Documents\Python projects\snowflake_postgres_elt_pipeline\venv\lib\site-packages\dbt\include\starter_project
[0m12:54:24.069599 [info ] [MainThread]: Profile dbt_project written to C:\Users\HP\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m12:54:24.069599 [info ] [MainThread]: 
Your new dbt project "dbt_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m12:54:24.085602 [debug] [MainThread]: Command `dbt init` succeeded at 12:54:24.085602 after 399.83 seconds
[0m12:54:24.085602 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E2691B5EA0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E26B925600>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E26B925660>]}
[0m12:54:24.085602 [debug] [MainThread]: Flushing usage events
[0m20:29:17.286136 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001ADFD455EA0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001ADFFBC56F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001ADFFBC5540>]}


============================== 20:29:17.302136 | f1138ab0-7065-448c-9bb1-ad28b15f8f39 ==============================
[0m20:29:17.302136 [info ] [MainThread]: Running with dbt=1.5.0
[0m20:29:17.302136 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': 'C:\\Users\\HP\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'send_anonymous_usage_stats': 'True'}
[0m20:29:17.302136 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\HP\OneDrive\Documents\Python projects\snowflake_postgres_elt_pipeline\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m20:29:17.310138 [debug] [MainThread]: Command `dbt run` failed at 20:29:17.310138 after 0.28 seconds
[0m20:29:17.310138 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001ADFD455EA0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001ADFFBC6110>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001ADFFBC5810>]}
[0m20:29:17.310138 [debug] [MainThread]: Flushing usage events
